git_greppers/local-git-setup.sh

```
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# This script sets up local git repositories for generating update files
+# It runs git_grepper.sh against multiple repositories and then runs the update generation
+# Author: Based on repositories used by lazrossi's setup
+
+# Get the directory of this script
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+cd "$SCRIPT_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)